doc/sphinx/changes.rst

```diff
@@ -14,6 +14,134 @@ Version 9.0
 Summary of changes
 ~~~~~~~~~~~~~~~~~~
 
+The Rocq Prover version 9.0 is the first Rocq Prover release after the renaming
+from The Coq Proof Assistant. The Rocq Prover 9.0 is backwards compatible with
```

Comment:
maybe say "the command line" is backwards compatible since I think we do have compatibility breaking changes otherwise